python/README.md

```diff
@@ -0,0 +1,34 @@
+# Python CSFLE Example
+
+## Steps
+
+1. Start a locally running `mongod` version >= 4.2 running on port 27017
+2. It is recommended you create a local environment to encapsulate the
+   dependencies. We use **pyenv**
```

Comment:
Nitpick: a link may be useful since `pyenv` is not part of a python base install.